src/main/java/mobile/common/AbstractCommonPage.java

```diff
@@ -0,0 +1,14 @@
+package mobile.common;
+
+import com.zebrunner.carina.webdriver.gui.AbstractPage;
+import org.openqa.selenium.WebDriver;
+
+public abstract class AbstractCommonPage extends AbstractPage {
```

Comment:
```suggestion
public abstract class AbstractCommonPage extends AbstractPage implements IMobileUtils {
```
after this change please remove `implements IMobileUtils` from all child classes